Gemfile

```diff
@@ -7,6 +7,9 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
+# Simple Command
```

Comment:
I know what it is - I'd prefer the comment explain why we added the gem. For instance "Simple Command is a gem that makes it easy to produce nice command-patterns"